### PR TITLE
Fix description of awardCriteriaDetails (remove 'selection) #1071

### LINKED ISSUE #1071: Fix description of awardCriteriaDetails
awardCriteriaDetails is described as:

> Any detailed or further information on the award or selection criteria.

However, as mentioned in #162, award criteria are not selection criteria. Previous to #162, the description was:

> Any detailed or further information on the selection criteria.

So, "award or" was added. The correct change would have been to remove "selection".

Originally identified in https://github.com/open-contracting/standard/issues/901#issuecomment-693584325

### PR DESCRIPTION
**Pull request by ColinMaudry**
Closes #1071

### CODE REVIEW

**Comment by ColinMaudry** [comment]
@jpmckinney I already had the issue, but I can't remember what the solution was. Something about cloning a different repo?

My `.git/config`:

```
[core]
	repositoryformatversion = 0
	filemode = true
	bare = false
	logallrefupdates = true
[remote "origin"]
	url = git@github.com:open-contracting/standard
	fetch = +refs/heads/*:refs/remotes/origin/*
[branch "1.1-dev"]
	remote = origin
	merge = refs/heads/1.1-dev
[branch "1.2-dev"]
	remote = origin
	merge = refs/heads/1.2-dev

```

**Comment by jpmckinney** [comment]
@ColinMaudry What's your issue? To get the PR to pass, just run `python util/make_dereferenced_release_schema.py` as documented: https://ocds-standard-development-handbook.readthedocs.io/en/latest/standard/contributing.html#proposing-changes

**Comment by jpmckinney** [comment]
Please be sure to follow all steps there (changelog, etc.).

File: docs/history/changelog.md [hunk 1]
```diff
@@ -36,6 +36,7 @@ Per the [normative and non-normative content and changes policy](https://docs.go
 
 * [#1086](https://github.com/open-contracting/standard/pull/1086) Recommend the use of strings rather than integers for identifiers.
 * [#1094](https://github.com/open-contracting/standard/pull/1094) Clarify the uniqueness of `Organization.id`
+* [#1109](https://github.com/open-contracting/standard/pull/1109) Disambiguate the description of `Tender.awardCriteriaDetails`
```
Comment (jpmckinney) [inline docs/history/changelog.md]:
```suggestion
* [#1109](https://github.com/open-contracting/standard/pull/1109) Remove "selection criteria" from the description of `Tender.awardCriteriaDetails`.
```

**Pull request merged**